src/main/java/com/rackspace/salus/event/ingest/services/KapacitorConnectionPool.java

```diff
@@ -12,38 +12,51 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
  */
 
 package com.rackspace.salus.event.ingest.services;
 
 import com.rackspace.salus.event.discovery.EngineInstance;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
 import java.io.Closeable;
-import java.io.IOException;
 import java.util.concurrent.ConcurrentHashMap;
+import lombok.extern.slf4j.Slf4j;
 import org.influxdb.BatchOptions;
 import org.influxdb.InfluxDB;
 import org.influxdb.InfluxDBFactory;
 import org.springframework.stereotype.Service;
 
 @Service
+@Slf4j
 public class KapacitorConnectionPool implements Closeable {
   private final ConcurrentHashMap<EngineInstance, InfluxDB> influxConnections =
       new ConcurrentHashMap<>();
 
+  private final Counter batchIngestFailure;
+
+  public KapacitorConnectionPool(MeterRegistry meterRegistry) {
+    this.batchIngestFailure = meterRegistry.counter("errors","operation", "batchFailure");
```

Comment:
Now that I'm looking at this closer, spelling out the operation would be helpful in grafana
```suggestion
    this.batchIngestFailure = meterRegistry.counter("errors","operation", "batchIngestFailure");
```